integration-tests/create-react-app/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "create-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1"
+  },
+  "optionalDependencies": {
+    "react-scripts": "1.0.11"
+  },
+  "devDependencies": {
+    "@percy-io/react-percy": "^0.1.5",
+    "webpack": "^3.5.5"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject",
+    "percy": "react-percy"
```

Comment:
I made our webpack config resolver a bit smarter as part of this. Previously we required users to always pass in an explicit webpack config path. Now, if no path is specified we check for a `webpack.config.js` file in the package root (which is what webpack also does). And we also now auto-detect `create-react-app` projects just like storybook, so we can grab the webpack config for those as well.